### ISSUE TITLE
Why is the lable of positive samples with two sentences similar is 0 ?

### ISSUE THREAD

**Issue by FirstDiscoverer** [udel-biotm-lab/BERT-CLRE#2]
`
def create_model_contrastive_pretraining()

        for positives in [zis, zjs]:
            l_neg = _dot_simililarity_dim2(positives, negatives)  # (64, 128)        (64, 768) (128, 768)

            labels = tf.zeros(int(FLAGS.train_batch_size / 2), dtype=tf.int32)  # (64)

            l_neg = tf.boolean_mask(l_neg, negative_mask)  # (64 * （128-2))     (64, 128) (64, 128)  TODO
            l_neg = tf.reshape(l_neg, (int(FLAGS.train_batch_size / 2), -1))  # (64, 128-2)
            l_neg /= 0.1

            logits = tf.concat([l_pos, l_neg], axis=1)  # [N,K+1]  
            loss += criterion(y_pred=logits, y_true=labels)       #
`

I think `l_pos` is a positive sample where two texts are similar, why is the label not 1
